saskatoon/sitebase/templatetags/pagination.py

```diff
@@ -0,0 +1,47 @@
+from django import template
+from rest_framework.utils.urls import remove_query_param, replace_query_param
+from saskatoon.pagination import BasicPageNumberPagination
+
+register = template.Library()
+page_query_param = BasicPageNumberPagination.page_query_param
+page_size_query_param = BasicPageNumberPagination.page_size_query_param
```

Comment:
how about importing these two values like this ?
```
from saskatoon.pagination.BasicPageNumberPagination import page_query_param, page_size_query_param
```

Reply:
it refuses to import it like that,
it says `'saskatoon.pagination' is not a package` althought the saskatoon folder has `__init__.py`

full exception:
```sh
django.template.library.InvalidTemplateLibrary: Invalid template library specified. ImportError raised when trying to load 'sitebase.templatetags.pagination': No module named 'saskatoon.pagination.BasicPageNumberPagination'; 'saskatoon.pagination' is not a package
```

Reply:
ok my bad!!